tests/unit/parallel/executors/minimal_async_executor.cpp

```diff
@@ -91,7 +92,8 @@ void test_bulk_async(Executor& exec)
     using hpx::util::placeholders::_1;
 
     typedef hpx::parallel::executor_traits<Executor> traits;
-    traits::async_execute(exec, hpx::util::bind(&async_bulk_test, tid, _1), v).get();
+    hpx::wait_all(traits::async_execute(
+        exec, hpx::util::bind(&async_bulk_test, tid, _1), v));
```

Comment:
This will not throw exceptions, if needed. `wait_all` does not try to retrieve the values out of the futures.